Optim-wip: Correct weight visualization citation

ProGamerGov opened the pull request:
* Fixed the link in `extract_expanded_weights` so that it now correctly points to the research article: https://distill.pub/2020/circuits/visualizing-weights/

* Corrected the first author name to be Chelsea Voss (@csvoss) in the `extract_expanded_weights` citation.

Should I also add a link to the article in the notebook?

NarineK commented:
Thank you, @ProGamerGov! Sure, feel free to update it in the notebook as well.

ProGamerGov commented:
@NarineK I've updated the tutorial to link to the _Visualizing Weights_ article, and I also replaced the high-low frequency detectors "forthcoming paper..." with a link to that section's published article as well!

NarineK reviewed (approved):
LGTM! Thank you :)
